internal/agent/device/hook/default_hooks.go

```diff
@@ -0,0 +1,94 @@
+package hook
+
+import (
+	"github.com/flightctl/flightctl/api/v1alpha1"
+	"github.com/samber/lo"
+)
+
+func marshalExecutable(run string, envVars *[]string, workDir string, timeout string, on []v1alpha1.FileOperation) v1alpha1.HookAction {
+	cha := v1alpha1.HookActionExecutableSpec{
+		Executable: v1alpha1.HookActionExecutable{
+			EnvVars: envVars,
+			Run:     run,
+			WorkDir: &workDir,
+		},
+		Timeout: &timeout,
+		On:      &on,
+	}
+	var ret v1alpha1.HookAction
+	_ = ret.FromHookActionExecutableSpec(cha)
+	return ret
+}
+
+func defaultAfterUpdateHooks() []v1alpha1.DeviceHookSpec {
+	return []v1alpha1.DeviceHookSpec{
+		{
+			Name:        lo.ToPtr("docker compose up"),
+			Path:        lo.ToPtr("/var/run/flightctl/compose/docker"),
+			Description: lo.ToPtr("Bring up a multi-container system based on the provided YAML docker-compose definition file"),
+			Actions: []v1alpha1.HookAction{
+				marshalExecutable("docker-compose -f {{ .FilePath }} up -d", lo.ToPtr([]string{"DOCKER_HOST=unix:///run/podman/podman.sock"}),
+					"/var/run/flightctl/compose/docker", "1m", []v1alpha1.FileOperation{v1alpha1.FileOperationCreate, v1alpha1.FileOperationUpdate}),
+			},
+		},
+		{
+			Name:        lo.ToPtr("podman compose up"),
+			Path:        lo.ToPtr("/var/run/flightctl/compose/podman"),
+			Description: lo.ToPtr("Bring up a multi-container system based on the provided YAML podman-compose definition file"),
+			Actions: []v1alpha1.HookAction{
+				marshalExecutable("podman-compose -f {{ .FilePath }} up -d", nil,
+					"/var/run/flightctl/compose/podman", "1m", []v1alpha1.FileOperation{v1alpha1.FileOperationCreate, v1alpha1.FileOperationUpdate}),
+			},
+		},
+		{
+			Name:        lo.ToPtr("quadlet up"),
```

Comment:
I really appreciate you did this work but I want to drop quadlet support for the demo. its going to be enough to provide examples of docker-compose and ensure its stabilized. We will return it soon.

Reply:
Done